timescaledb/how-to-guides/replication-and-ha/configure-replication.md

```diff
@@ -163,37 +163,35 @@ from a base backup of the primary instance.
     ```
     The -W flag prompts you for a password. If you are using this command in an
     automated setup, you might need to use a [pgpass file][pgpass-file].
-1.  When the backup is complete, create a [recovery.conf][postgres-recovery-docs]
+1.  When the backup is complete, create a [standby.signal][postgres-recovery-docs]
     file in your data directory, and set the appropriate permissions. 
```

Comment:
What are the appropriate permissions?

Reply:
Ah, it is given below, but might be good to write here as well.

Reply:
Absolutely! Thanks @mkindahl. Just want to double check with you that `0600` is the appropriate permissions level? Those were the permissions for `recovery.conf`, but after trawling the PostgreSQL docs, I've seen nothing about whether that's still the case for `standby.signal`.